.github/workflows/production.yaml

```
@@ -0,0 +1,70 @@
+      - name: Install Dependencies
+        run: npm ci --force
+
+      - name: Run Migrations
+        run: npm run prisma:deploy
+
+  deploy:
+    name: Deployment
+    needs: [release, migrations]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)